main.js

```diff
@@ -124,8 +130,38 @@ for (var i = 0 ; i < courses.length ; i++) {
    }
 }
 
+sortNameDown.style.display="none";
+withoutSort.style.display="none";
+sortName.style.display= "inline-block" ;
+
+function toggleSortOrder() {
+ 
+
+    if (sortName.style.display == "inline-block") {
+        sortElement = courses.slice();
+        courses.sort(function(a, b) {
+            return a.name.localeCompare(b.name);
+        });
+        sortName.style.display = "none";
+        sortNameDown.style.display = "inline-block";
+    } else {
+        courses.sort(function(a, b) {
+            return b.name.localeCompare(a.name);
+        });
+        sortNameDown.style.display = "none";
+        withoutSort.style.display="inline-block";
 
+    }
 
-
-
-
+    display(courses);
+}
+// to display course without sort 
+withoutSort.onclick = function(){  
+    courses=sortElement ;
+     display();
+     sortName.style.display="inline-block";
+     withoutSort.style.display= "none";
+}
+// to display function toggleSortOrder when we click at sort/sortdowm 
+sortName.onclick = toggleSortOrder;
```

Comment:
can you add the onClick event from the html attribute instead of attaching it here ?